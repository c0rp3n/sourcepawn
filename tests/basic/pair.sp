#include <shell>

Pair g_oPair = null;

public void main()
{
    g_oPair = new Pair(32, 4.0);
    print("first: "); printnum(g_oPair.First());
    print("second: "); printfloat(g_oPair.Second());
    delete g_oPair;

    print("\n");

    g_oPair = new Pair(3.0, 24);
    print("first: "); printfloat(g_oPair.First());
    print("second: "); printnum(g_oPair.Second());
    delete g_oPair;
}