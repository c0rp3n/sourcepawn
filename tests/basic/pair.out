first: 32
second: 4.0

first: 3.0
second: 24